content/en/tracing/trace_collection/custom_instrumentation/go/migration.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Migrating from v1 to v2 of the Go Tracer
+description: 'Upgrade your Go tracer from v1 to v2.'
+further_reading:
+    - link: 'tracing/trace_collection/custom_instrumentation/dd-api'
+      tag: 'Documentation'
+      text: 'Get started with v1 of the Go Tracer'
+    - link: 'tracing/trace_collection/custom_instrumentation/dd-api-v2'
+      tag: 'Documentation'
+      text: 'Get started with v2 of the Go Tracer'
+---
+
+<div class="alert alert-info">This documentation assumes that you are currently using version v1.x of the Go Tracer. If you are already on v2.x of the tracer, please visit <a href="tracing/trace_collection/custom_instrumentation/dd-api-v2">Go v2 Custom Instrumentation using the Datadog API</a> documentation instead.</div>
```

Comment:
```suggestion
<div class="alert alert-info">This documentation assumes that you are using version v1.x of the Go tracer. If you are using v2.x, see <a href="tracing/trace_collection/custom_instrumentation/dd-api">Go Custom Instrumentation using the Datadog API</a> instead.</div>
```
Note, the file the link was referencing initially doesn't exist, so I pointed to the custom instrumentation doc that has v1 and v2.